### PR TITLE
WHACK-22: Remove Jetty

### PR DESCRIPTION
**Pull request by guusdk**
The first commit in this PR is utterly useless, but I kinda like that it's been waiting to be merged since 2016. :)

### CODE REVIEW

**Comment by guusdk** [comment]
I've not tested this change, other than that it compiles.

File: source/java/org/jivesoftware/whack/container/ServerContainer.java [hunk 1]
```diff
@@ -1,9 +1,6 @@
 /**
- * $RCSfile$
- * $Revision$
- * $Date$
  *
- * Copyright 2005 Jive Software.
+ * Copyright 2005 Jive Softwar, 2024 Ignite Realtime Foundation
```
Comment (Fishbowler) [inline source/java/org/jivesoftware/whack/container/ServerContainer.java:3]:
```suggestion
 * Copyright 2005 Jive Software, 2024 Ignite Realtime Foundation
```

File: source/pom.xml [hunk 2]
```diff
@@ -41,14 +41,14 @@
 		<dependency>
 			<groupId>org.eclipse.jetty</groupId>
 			<artifactId>jetty-server</artifactId>
-			<version>7.0.1.v20091125</version>
```
Comment (Fishbowler) [inline source/pom.xml:44]: This version number includes suitable shaming for how long it's been 😂

**Review by Fishbowler — APPROVED** [review]
One minor nit, but LGTM

**Comment by guusdk** [comment]
I've reconsidered this PR. Lets _remove_ Jetty, rather than upgrade it. Jetty is a somewhat heavy/complex dependency, that goes mostly unused.

This comment was added to the source code _in 2008_:

> Package not being currently used.
> 
> This package was created with the idea that Whack could be
> run as a web application running in a webserver (e.g. Jetty).  It would support plugins that
> could be dynamically dropped and the server would detect them and use them without a restart.
> The plugins could be new external components that were added to the web app. Each external
> component could extend the web app and add new configuration pages so that admins could
> easily configure the plugins/external components.
> 
> _Note that conf/jive-whack.xml, security folder and web folder are also not being used._

Let's just remove Jetty, instead of carrying over technical debt.

We can leave the commit that (attempts to) upgrade Jetty from 7 to 12 in, as a starting point for anyone interested in retaining it after all, but I'd be surprised if anyone actually is. If you're reading this and are bothered by Jetty now being gone, please let us know!

**Review by Fishbowler — APPROVED** [review]

**Pull request merged**